src/improvements/addresses.toml

```diff
@@ -8,3 +8,11 @@ SecurityCouncil = "0xc2819DC788505Aac350142A7A707BF9D03E3Bd03"
 FoundationOperationSafe = "0x837DE453AD5F21E89771e3c06239d8236c0EFd5E"
 FoundationUpgradeSafe = "0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B"
 SecurityCouncil = "0xf64bc17485f0B4Ea5F06A96514182FC4cB561977"
+# These extra contracts (specifically used by the OPCMUpgradeV200 template) are needed here
```

Comment:
Am I remembering correctly, did we say that said task specific addresses should be added to the `config.toml` of the task and the address registry should be clever enough to pick these up?

Reply:
Right now, we have task level addresses in `addresses.toml` which isn't clean and won't scale.